ui/imports/utils/Utils.qml

```diff
@@ -796,6 +796,17 @@ QtObject {
         }
     }
 
+    function getFontSizeBasedOnLetters(text) {
```

Comment:
@Khushboo-dev-cpp have you tried something like this instead?
https://doc.qt.io/qt-5/qml-qtquick-text.html#fontSizeMode-prop

Reply:
nope, let me try it out, thanks @saledjenic

Reply:
wait in this case i dont think it will work causes its a TextEdit inside

Reply:
This looks like real magic, `getFontSizeBasedOnLetterCount` might be the appropriate naming. Hard to define its general purpose, but I assume it is here for sharing as imports between different places.

This can be better achieved using [FontMetrics](https://doc.qt.io/qt-5/qml-qtquick-fontmetrics.html), more exactly the [maximumCharacterWidth](https://doc.qt.io/qt-5/qml-qtquick-fontmetrics.html#maximumCharacterWidth-prop) against a certain width. However, hard to justify if this works reliably.